modules/service-fips.sh

```diff
@@ -1,5 +1,6 @@
-# shellcheck disable=SC2039
+# shellcheck disable=SC2034,SC2039
 
+FIPS_SERVICE_NAME="Canonical FIPS 140-2 Modules"
```

Comment:
I think these variables <service>_SERVICE_NAME should be called <service>_SERVICE_TITLE, because we are already using "esm", "livepatch" and "fips" as names elsewhere.

Reply:
Done